packages/cli/src/databases/repositories/test-run.repository.ee.ts

```diff
@@ -29,6 +29,10 @@ export class TestRunRepository extends Repository<TestRun> {
 		return await this.update(id, { status: 'completed', completedAt: new Date(), metrics });
 	}
 
+	public async markAsCancelled(id: string) {
+		return await this.update(id, { status: 'cancelled' });
```

Comment:
If we cancel a test run that is in state `new`, it won't have the count fields set and this will fail the constraint checks

Reply:
Keen eye! I just updated the constraint check in another branch to account for this :)